templates/medium/cxf-contract-first/medium-fuse-cxf-contract-first-java/pom.xml

```
@@ -0,0 +1,283 @@
+						<phase>generate-sources</phase>
+						<configuration>
+							<sourceRoot>target/generated/src/main/java</sourceRoot>
+							<wsdlOptions>
+								<wsdlOption>
+									<wsdl>src/main/resources/wsdl/order.wsdl</wsdl>
+									<extraargs>
+										<extraarg>-impl</extraarg>
+									</extraargs>
+								</wsdlOption>
+							</wsdlOptions>
+							<additionalJvmArgs>-Djavax.xml.accessExternalSchema=jar:file,file</additionalJvmArgs>
+						</configuration>
+						<goals>
+							<goal>wsdl2java</goal>
+						</goals>
+					</execution>
+				</executions>
+			</plugin>
+
+
+			<!-- Compiler plugin enforces Java 1.7 compatibility and activates annotation 
+				processors -->
+			<plugin>
+				<artifactId>maven-compiler-plugin</artifactId>
+				<version>3.1</version>
+				<configuration>
+					<source>1.7</source>
+					<target>1.7</target>
+				</configuration>
+			</plugin>
+
+			<plugin>
+				<groupId>org.apache.felix</groupId>
+				<artifactId>maven-bundle-plugin</artifactId>
+				<version>${version.maven-bundle-plugin}</version>
+				<extensions>true</extensions>
+				<executions>
+          			<execution>
+            			<id>bundle-manifest</id>
+           				<phase>process-classes</phase>
+            			<goals>
+              				<goal>manifest</goal>
+            			</goals>
+          			</execution>
+        		</executions>
+				<configuration>
+					<instructions>
+						<Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+						<Export-Package>ws.camel.mycompany.com</Export-Package>
+						<Import-Package>*</Import-Package>
+					</instructions>
+				</configuration>
+			</plugin>
+
+			<!-- this plugin will use the fabric.* properties to configure its behaviour 
+				see the Properties section here: http://fabric8.io/gitbook/mavenPlugin.html -->
+			<plugin>
+				<groupId>io.fabric8</groupId>
+				<artifactId>fabric8-maven-plugin</artifactId>
+				<version>1.2.0.redhat-621084</version>
+			</plugin>
+		</plugins>
+
```

we should add m2e lifecycle mapping to ignore errors from the felix plugin. see the other templates which are finished.